src/v2/view-builder/internals/BaseView.js

```diff
@@ -34,6 +35,31 @@ export default View.extend({
       },
     });
     this.add(this.Footer, { selector : '.siw-main-footer' });
+
+    // add callout for messages
+    this.messages = options.messages;
+    if (this.messages && this.messages.value.length) {
+      this.showMessageCallout(options.messages.value[0].message, 'warning');
```

Comment:
`this.messages` was created but never used since you're referring `options.messages`.